Utils/Elasticsearch/Oracle2JSON.py

```diff
@@ -85,6 +86,39 @@ def squash(conn, queries, offset_date, end_date):
                                    if row['type'] == 'output' and row['taskid'] == task['taskid']]
         sys.stdout.write(json.dumps(task) + '\n')
 
+
+def squash_records(rec):
+    """
+    a single-pass iterator (for a generator) restructuring list of datasets: 
+    from view: 
+    [
```

Comment:
/* Just a comment ;) */
The function was added, but not used -- so it is not obvious, what was the purpose, before one reads later commits, as for now nothing in the program behaviour has changed.

And vice versa, in the later commits the real scope of the changes is somehow hidden, masked: the new function is called, but how different it is from what was before? was some code just moved to a separate function, or changed completely?